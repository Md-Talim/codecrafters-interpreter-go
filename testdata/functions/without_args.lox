// Test Case 1
fun foo() {
  print 7;
}
foo();

// Test Case 2
// Prints nothing
fun f() {}
f();

// Test Case 3
fun foo() {}
print foo;

// Test Case 4
fun cumulative_sum() {
    var n = 10;  // Fixed value
    var total = 0;
    var i = 1;
    while (i <= n) {
        total = total + i;
        i = i + 1;
    }
    print "The cumulative sum from 1 to 10 is:";
    print total;
}

cumulative_sum();