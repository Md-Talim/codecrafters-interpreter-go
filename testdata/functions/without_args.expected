7
<fn foo>
The cumulative sum from 1 to 10 is:
55